plugin/MyFirstJavaPlugin/src/main/java/com/github/yournamehere/MyFirstJavaPlugin.java

```diff
@@ -0,0 +1,117 @@
+package com.github.yournamehere;
+
+import android.content.Context;
+
+import com.aliucord.Utils;
+import com.aliucord.annotations.AliucordPlugin;
+import com.aliucord.api.CommandsAPI;
+import com.aliucord.entities.MessageEmbedBuilder;
+import com.aliucord.entities.Plugin;
+import com.aliucord.patcher.Hook;
+import com.aliucord.patcher.InsteadHook;
+import com.aliucord.patcher.PreHook;
+import com.aliucord.wrappers.embeds.MessageEmbedWrapper;
+import com.discord.api.commands.ApplicationCommandType;
+import com.discord.models.user.CoreUser;
+import com.discord.stores.StoreUserTyping;
+import com.discord.widgets.chat.list.adapter.WidgetChatListAdapterItemMessage;
+import com.discord.widgets.chat.list.entries.ChatListEntry;
+import com.discord.widgets.chat.list.entries.MessageEntry;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+// Aliucord Plugin annotation. Must be present on the main class of your plugin
+// Plugin class. Must extend Plugin and override start and stop
+// Learn more: https://github.com/Aliucord/documentation/blob/main/plugin-dev/1_introduction.md#basic-plugin-structure
+@AliucordPlugin(
+        requiresRestart = false // Whether your plugin requires a restart after being installed/updated
+)
+class MyFirstJavaPlugin extends Plugin {
```

Comment:
```suggestion
public class MyFirstJavaPlugin extends Plugin {
```
this wouldn't work without public for me